### DISCUSSION TITLE
Você vive, viveu ou viveria como Nômade Digital? Conte mais 😄

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#1996] [Discussão]
Fiz essa pergunta no [Twitter e tiveram respostas sensacionais com algumas experiências](https://twitter.com/felipefialho_/status/1354809152711745541) (vale ler, sério!).

Como percebi que tem muita gente interessada no tema e muita gente que vive dessa forma, achei uma boa ideia criar essa discussão pra deixar esse papo mais estruturado 😁

- Quais os pontos positivos? 
- Quais os pontos negativos? 
- Quais sites ou aplicativos auxiliam nessa vida? 
- Quais dicas pra quem tem interesse?

**Comment by willianjusten** [comment]
Eu fui nômade digital por aproximadamente 1 ano, não por planejamento, mas por consequências da vida. Eu estava morando na Irlanda, arrumei um emprego na Toptal e acabei viajando a Europa sem rumo. Nisso eu pude notar algumas coisas:

## Pontos positivos

Alguns pontos positivos vão de encontro com o trabalho remoto, mas que querendo ou não, entram junto também.

- Poder trabalhar de onde quiser
- Não ter preocupações de translado para o trabalho
- Poder vivenciar diferentes culturas (tanto em outros países ou nem precisa ir longe, até interiores do próprio Brasil)
- Aprender a se desapegar das coisas materiais (já que você não vai poder se locomover com mt coisa)
- Aprender a conviver melhor com as pessoas (a não ser que seu budget seja bom, pode ser que você passe dias em hostels, então vai aprender a dividir ambientes, até mesmo em alguns Airbnbs)
- Sensação de liberdade, se não gostar de lugar X, simplesmente se mudar

## Pontos negativos

- Sensação de "não pertencimento", acaba que você nunca se sente 100% "no seu lar"
- Trabalhar em qq lugar é legal, mas com o tempo a coluna sente, nem sempre tem um Airbnb bom com mesa/cadeira ou lugar quieto. Isso pode ser bastante ruim se precisar focar em tarefas bem complexas.
- Muitos lugares podem ter internet péssima!
- Viajar para diferentes países é legal, mas precisa lidar com burocracia de visto/imigração
- Não poder ter animais/plantas
- Saudades de amigos/família. Várias vezes você está feliz em algum lugar, mas gostaria de estar compartilhando com aquele amigo.

## Sites/Aplicativos

- Os clássicos Airbnb, Trivago/Booking, TripAdvisor para achar cantos para ficar
- O [NomadList](https://nomadlist.com/) tem bastante coisa interessante
- O [Worldpackers](https://www.worldpackers.com/) tem várias dicas e guias
- Qualquer app de controle financeiro como Expensify ou outro (vai por mim, é importante ter noção dos seus gastos)
- Utilize um cartão pré-pago para fugir das taxas como a [Payoneer](https://www.payoneer.com/) ou [Transferwise](https://transferwise.com/br)

## Dicas

- Faça planejamentos de gastos, locais para ir, até mesmo se o lugar vai ter internet/lugar para trabalhar direito
- Organize seu dia para que você não só trabalhe ou só fique passeando, equilíbrio é importante, senão você pode virar vagabundo no trabalho ou pode nem aproveitar sua estadia.
- Evite se locomover tão rápido. Tem gente que passa 2-3 dias em cidade X, já pula para outra e assim vai. Se você estiver trabalhando, 2-3 dias não são suficientes para poder conhecer as coisas e trabalhar, então aumente alguns dias.
- Tenha sempre um dinheiro reserva de segurança e faça seguro! Pode parecer que tá jogando dinheiro fora com seguro, mas tenha certeza que o seguro vai te salvar em milhares de oportunidades!
- Tente criar uma renda passiva, seja com investimentos, venda de produto, enfim. Isso vai te aliviar da preocupação de perder seu emprego, o que pode deixar nervoso e aproveitar menos os passeios.
- Cuide da sua saúde física e principalmente da mental, viajar sozinho é um desafio, parece um paraíso, mas tem momentos de solidão e tristeza, saiba controlar isso.

---

Bom, essas são algumas coisas que eu pude experienciar. Na minha opinião, todo mundo deveria tentar, nem que só por 1 ano (como meu caso), é algo maravilhoso, como acima, tem seus prós e contras, mas é uma história de vida para compartilhar com amigos e filhos depois xD

**Comment by jonatasbaldin** [comment]
Fiz uma thread no Twitter sobre isso heh

https://threadreaderapp.com/thread/1355171384108920833.html
